<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvContainerSearch</class>
 <widget class="QDialog" name="EnvContainerSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="m_searchBarLayout">
     <item>
      <widget class="QLineEdit" name="m_searchLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="m_itemsView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_searchButtonsLayout">
     <item>
      <widget class="QPushButton" name="m_addItemButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
